.github/workflows/test-n-publish.yml

```diff
@@ -37,11 +37,11 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: '3.9'
+        python-version: '3.8'
```

Comment:
Do we want a matrix here?  I wasn't sure. Maybe we can address that later.

Reply:
3.8 is the lowest version we support so we should make sure the test run at least on that platform. Not sure either if we want a matrix here :) I've a feeling most user are on 3.9.